xcm-chain-registry-builder/dist/transformed-chain-registry.json

```diff
@@ -0,0 +1,3727 @@
+        }
+      ],
+      "2031": [
+        {
+          "assetId": "13",
+          "symbol": "CFG",
+          "decimals": 18,
+          "xcmLocation": {
+            "v1": {
+              "parents": 1,
+              "interior": {
+                "x2": [
+                  {
+                    "parachain": 2031
+                  },
+                  {
+                    "generalKey": "0x0001"
+                  }
+                ]
+              }
+            }
+          },
+          "reserveType": "foreign",
+          "icon": "https://dev.sw-chain-list-assets.pages.dev/assets/chain-assets/centrifuge-native-cfg.png"
+        }
+      ],
+      "2032": [
+        {
+          "assetId": "11",
+          "symbol": "iBTC",
+          "decimals": 8,
+          "xcmLocation": {
+            "v1": {
+              "parents": 1,
+              "interior": {
+                "x2": [
+                  {
+                    "parachain": 2032
+                  },
+                  {
+                    "generalKey": "0x0001"
+                  }
+                ]
+              }
+            }
+          },
+          "reserveType": "foreign",
+          "icon": "https://dev.sw-chain-list-assets.pages.dev/assets/chain-assets/moonbeam-local-xcibtc.png"
+        },
+        {
+          "assetId": "17",
+          "symbol": "INTR",
+          "decimals": 10,
+          "xcmLocation": {
+            "v1": {
+              "parents": 1,
+              "interior": {
+                "x2": [
+                  {
+                    "parachain": 2032
+                  },
+                  {
+                    "generalKey": "0x0002"
+                  }
+                ]
+              }
+            }
+          },
+          "reserveType": "foreign",
+          "icon": "https://dev.sw-chain-list-assets.pages.dev/assets/chain-assets/moonbeam-local-xcintr.png"
+        }
+      ],
+      "2035": [
+        {
+          "assetId": "8",
+          "symbol": "PHA",
+          "decimals": 12,
+          "xcmLocation": {
+            "v1": {
+              "parents": 1,
+              "interior": {
+                "x1": {
+                  "parachain": 2035
+                }
+              }
+            }
+          },
+          "reserveType": "foreign",
+          "icon": "https://dev.sw-chain-list-assets.pages.dev/assets/chain-assets/phala-native-pha.png"
+        }
+      ],
+      "2037": [
+        {
+          "assetId": "25",
+          "symbol": "UNQ",
+          "decimals": 18,
+          "xcmLocation": {
+            "v1": {
+              "parents": 1,
+              "interior": {
+                "x1": {
+                  "parachain": 2037
+                }
+              }
+            }
+          },
+          "reserveType": "foreign",
+          "icon": "https://dev.sw-chain-list-assets.pages.dev/assets/chain-assets/unique_network-native-unq.png"
+        }
+      ],
+      "2046": [
```

Comment:
For the tokens in the chains that do not support XcmPaymentApi, xcAssetData can be simplified by removing this part of the data.